openmc/mgxs/mgxs.py

```diff
@@ -254,7 +301,11 @@ def scores(self):
     def filters(self):
         group_edges = self.energy_groups.group_edges
         energy_filter = openmc.EnergyFilter(group_edges)
-        return [[energy_filter]] * len(self.scores)
+        filters = []
```

Comment:
It was - but this makes the  `[energy_filters]` lists all be references of one. So when `_add_angle_filters` loops through each and adds the polar/azi filters, it ends up adding these filters `self.scores` times.